### PR TITLE
[Duplicate of #4238]: Track users using google analytics

### LINKED ISSUE #3778: Track Individual Logged in Users using Google Analytics
### Proposed title of the article
Track Individual Logged in Users using Google Analytics

### Introduction paragraph (2-3 paragraphs):
Google Analytics is a tool that helps us monitor the website's traffic. It collects information about the visitors who visit the site. Often people ask if it is really possible to track individual users in Google Analytics. And the answer is: It is possible if your application is a member-based application and has an authenticated mechanism.
For instance, if yours is a task management system or a shopping cart, Google Analytics can be used to monitor the activity of the users who are logged in.
In this article, we will demonstrate how to implement the same functionality of Google Analytics without the need for Google Tag Manager. Both tools are very different,in that Google tag manager is like a container for adding different tags. Wherein Google Analytics is one such tag that is used to collect and analyze your website traffic.
 

### Key takeaways: 
1. log in to Google Analytics and then create an Account.
2. Create Property.
3. Create View with User-ID enabled.
4. Create Custom Dimensions to track user types.
5. Modify tracking code to include user-id and custom dimensions.
6. Create custom reports, view and analyze data.

### References: 
N/A

### PR DESCRIPTION
**Pull request by Nancy245**
### Article checklist

- [X] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [X] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [X] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [X] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [X] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [X] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [X] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [X] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
This closes #3778

### CODE REVIEW

**Comment by Nancy245** [comment]
@srishilesh hello, could you check on this one

File: content/articles/track-idividual-logged-in-users-using-google-analytics/index.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+### Introduction
+Google Analytics is one of the most popular digital analytics software.
+It is Google’s free web analytics service that allows you to analyze in-depth detail about the visitors to your website.
+It provides valuable insights that can help you to shape the success strategy of your business.
```
Comment (srishilesh) [inline content/articles/track-idividual-logged-in-users-using-google-analytics/index.md:4]: A more briefer intro would be better.

Have a look at other articles to get an idea.

**Review by srishilesh — CHANGES_REQUESTED** [review]
@Nancy245 

The formatting looks good. It's ready to be reviewed.

Very minor suggestions:
- Make the intro a bit briefer. It's too small. Refer other articles.
- Every line is written in a new line. Club a few of them and make it as a paragraph.

Ping me when you're done, let's start the tech review.

Thanks.

**Comment by srishilesh** [comment]
Duplicate of #4238

**Comment by Nancy245** [comment]
@srishilesh, changes requested have been implemented

File: content/articles/track-idividual-logged-in-users-using-google-analytics/index.md (hunk 1 above)
Comment (srishilesh) [reply content/articles/track-idividual-logged-in-users-using-google-analytics/index.md]: @Nancy245 Don't think this has been done yet?

**Review by srishilesh — CHANGES_REQUESTED** [review]
@Nancy245 

Looks like you've missed out 1 suggestion.

Along with that you've another suggestion on the authors file.

Ping me when you're done.

To avoid closure of this PR, make sure every suggestion is covered.

Thanks.

**Comment by srishilesh** [comment]
@Nancy245 

Do we have any updates on this minor suggestion?

If you still need more time, we can definitely close this PR, and allow you to create a new PR with all the changes.

Let me know ASAP.

**Comment by Nancy245** [comment]
@srishilesh hello, let me work on the suggested changes I'll let you know when I'm done thanks

**Comment by Nancy245** [comment]
@srishilesh hello, I pushed some changes, and about the authors' file could you please explain what I should change or add to my author's file. Thank you

**Comment by hectorkambow** [comment]
> @srishilesh hello, let me work on the suggested changes I'll let you know when I'm done thanks

All changes should be completed (before submitting the PR and on the forked repo ) to avoid cluttering the backlog 👍 
FYI

**Comment by hectorkambow** [comment]
@Nancy245

**Comment by hectorkambow** [comment]
@Nancy245 

Good afternoon and thank you for submitting your topic suggestion. Not a problem but it seems that this PR is still quite a bit aways from being completed. 
 
In an effort to be fair to our larger EngEd community as a rule of thumb - we can only accept content (pull requests) that are complete and fully polished articles/tutorial - meaning they are ready to be published. 
 
All revisions and edits **should be completed on your own forked repo** (as to not take up room in the queue) - this allows all revisions to be done on your forked personal repo. 
 
We will be closing this PR and please re-submit it when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer and myself. Be sure to go through the entire article again to proofread - and fix any issues that the reviewer may have missed. 
 
Our Peer Reviewer and Content Moderator teams do NOT provide any revisions services. All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.
 
>NOTE: A duplicate PR will go back into the queue and will not receive any further major revisions. A duplicate PR should be 100% ready to be published, otherwise it will be closed immediately. We encourage our new contributors to go over our [resources page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) and [new contributor folder](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) to be better familiar with the level of quality and style we are looking to publish.
 
Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.

cc @srishilesh 
Please be sure to help the student via Slack 👍 with some tips, resources, and examples in our library 🚀  
We look forward to getting your EngEd portfolio set up and ready to share.

**Comment by srishilesh** [comment]
> @srishilesh hello, I pushed some changes, and about the authors' file could you please explain what I should change or add to my author's file. Thank you

@hectorkambow 

Looks like the author had pushed a commit with the suggested changes. Let me have a look at them, and close if any more revisions are needed.

Hence, reopening.

Cc. @Nancy245

**Comment by srishilesh** [comment]
Jumping on this for my final review.

**Comment by srishilesh** [comment]
@Nancy245 

I've a few questions for testing these code snippets. Pinging you for more info via Slack.

Until I get back with more info on the code snippets, I'll be closing this PR. Will reopen this once I'm good with the tech review.

cc. @hectorkambow

**Comment by srishilesh** [comment]
Pushing few of my changes here, before I close.

**Review by srishilesh — APPROVED** [review]
@Nancy245 

Good work!

The tech review took longer than expected. Though, you were helping me out via Slack, I couldn't find much documentation online.

Everything looks good now.

Make sure to go through my changes to understand and improve in your future articles.

Moving for the final review.

Thanks.

**Comment by lalith1403** [comment]
Jumping on this one 👍

**Review by lalith1403 — CHANGES_REQUESTED** [review]
There seems to be an error with rendering images. Can you check the URLs and the extension types? Additionally, shorten the URL of the article. You can change it to something like `track-users-google-analytics`.

Also make note of the changes made.

**Comment by hectorkambow** [comment]
@lalith1403 
Was the images rendering on that PR the only issues left to be fixed or does the article need more polishing?

**Comment by Nancy245** [comment]
@lalith1403 Hi, I have incorporated the necessary changes that you requested. Thank you

**Comment by srishilesh** [comment]
@lalith1403 

The requested changes has been made by the author. And, I've cross-verified the image URLs. Everything seems to be correct now, you may proceed with your review.

cc. @Nancy245

**Comment by hectorkambow** [comment]
checking this one out - to see image issue

**Pull request merged**